akaltsou/02_GIT.me

```diff
@@ -0,0 +1,50 @@
+  268  git init
```

Comment:
Could you please rename (.md) and add git markup

Reply:
done